### PR TITLE
Initial experience tweaks

### PR DESCRIPTION
**Pull request by danmcp**

### CODE REVIEW

**Comment by danmcp** [comment]
@pmorie Please review

File: Makefile [hunk 1]
```diff
@@ -10,7 +10,7 @@ linux:
 
 image: linux
 	cp servicebroker image/
-	docker build image/ -t quay.io/osb-starter-pack/servicebroker
+	sudo docker build image/ -t quay.io/osb-starter-pack/servicebroker
```
Comment (pmorie) [inline Makefile]: I would really prefer not to merge the sudo command; i’d be happy to have a link to how to add your user to the docker group, though

File: README.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ Open Service Broker.
 
 You'll need:
 
-- A running [kubernetes](https://github.com/kubernetes/kubernetes) cluster
+- A running [kubernetes](https://github.com/kubernetes/kubernetes) or [openshift](https://github.com/openshift/origin/) cluster
```
Comment (pmorie) [inline README.md]: Openshift is kubernetes

File: Makefile (hunk 1 above)
Comment (danmcp) [reply Makefile]: @pmorie I thought the official recommendation from Red Hat is to not add your user to the docker group.

https://www.projectatomic.io/blog/2015/08/why-we-dont-let-non-root-users-run-docker-in-centos-fedora-or-rhel/

File: README.md (hunk 2 above)
Comment (danmcp) [reply README.md]: You have specific instructions for openshift below.  Should I change this to say or openshift's distribution of kubernetes?

**Pull request merged**